docs/howtos/DataPlane Generation - DPG/05union.mdx

```diff
@@ -73,10 +88,12 @@ you must use `@sharedRoute`. By default, codegen will generate one method per op
 <TabItem value="typespec" label="TypeSpec" default>
 
 ```typespec
-@sharedRoute("/foo")
-op a(x: int): float;
+@sharedRoute
+@route("/foo")
+op a(x: int32): float;
 
-@sharedRoute("/foo")
+@sharedRoute
+@route("/foo")
 op b(x: string): int64;
 ```
```

Comment:
Update this typespec. The origin one is not valid.

https://typespec.io/docs/libraries/http/reference/decorators#@TypeSpec.Http.sharedRoute